python-frontend/src/main/java/org/sonar/python/types/DeclaredType.java

```diff
@@ -139,7 +145,7 @@ public String toString() {
   }
 
   public String typeName() {
```

Comment:
Not sure if this could break anything, but we should probably consider changing this to use the `pretty_printed_name` of Typeshed types here when available instead of recomputing them (not in this PR though...)

Reply:
Agreed, we can have a separate ticket for it

Reply:
Ticket created: https://jira.sonarsource.com/browse/SONARPY-880